pkg/diagnostics/analyzers.go

```diff
@@ -81,6 +81,10 @@ func (a *analyzerFactory) managementClusterDeploymentAnalyzers() []*Analyze {
 			Name:             "capv-controller-manager",
 			Namespace:        constants.CapvSystemNamespace,
 			ExpectedReplicas: 1,
+		}, {
```

Comment:
I dont think this will add the logs, this i believe, is just to validate the pod is running if the user runs the analyze subcommand

Reply:
no, but https://github.com/aws/eks-anywhere/pull/2201/files#diff-d32ef215cc246d7530a6ff1a2722c31aef67351081e996a8ab33545deded959cR72 should add the logs; when the cloudstack datacenter kind is passed in, it'll return the relevant collectors, which include collectors for the whole namespace. 

to extend the collectors associated with a cloudstack data center kind, check out: https://github.com/aws/eks-anywhere/blob/2b8e7f172428db6a328066d5ea07f5e544d2d6ad/pkg/diagnostics/collectors.go#L87